spec/unit/pops/lookup/context_spec.rb

```diff
@@ -171,13 +171,13 @@ module Lookup
 
       context 'and multiple compilations' do
 
-        before(:each) { Puppet.settings[:environment_timeout] = 'unlimited' }
         after(:each) do
           Puppet.settings[:environment_timeout] = 0
```

Comment:
Validated that is correct.  Having the code here has the positive of being more explicit though.

I'll update PR to remove that